modules/ROOT/pages/common-dev-strategies.adoc

```diff
@@ -0,0 +1,57 @@
+For details on the different transaction types, transaction actions operations and how you can configure a transaction in a message source or Try Scope in the xref:transaction-management.adoc[Implement Transaction Management] documentation. +
+Additionally, you can implement xref:single-resource-transaction.adoc[Single Source Transactions], xref:xa-transactions.adoc[XA Transactions] and xref:using-bitronix-to-manage-transactions.adoc[use Britonix to manage transactions].
+
+== Modularize Configurations Files
+Modularize configuration files by splitting monolithic configurations into several files, and leverage Mule’s capacity to load multiple configuration files at application start-up time. Although it might seem convenient to keep your Mule configurations in one file, a very large XML file quickly becomes unmanageable. Modularizing Mule configurations is an opportunity for splitting work across teams, whether these teams work on the same project or on different projects with an accent put on reuse. There are two options to load multiple configuration files:
+
+* Side-by-side: Provide a list of independent configuration files to load.
+* Imported: Have one configuration file import several others, which in turn, can import other files. +
+
+For details see xref:modularizing-your-configuration-files-for-team-development.adoc[Modularize Configurations Files] documentation.
+
+== Implement Continuous Integration
+Implement continuous integration (CI) for your projects by using Maven as your build tool. Create a build that gets triggered on every project change, and run all of its unit and functional tests automatically. By doing so you can:
+Get early notification of issues in the software development lifecycle.
+Ensure code gets fully tested before release.
+Confirm that successfully tested branches ensure better success when merging to the master branch. +
+For details see xref:continuous-integration.adoc[Continuous Integration] documentation.
+
+== Implement Orchestration in Mule
+Implement orchestration is Mule to facilitate the process of moving data through a Mule implementation based on flows. The orchestration implements business-level processes combining business-specific services across applications and information systems.
+Flows uses cases that orchestrate are:
+
+* Simple integration.
+* Scheduling data processing.
+* Connecting cloud and on-premise applications.
+* Event processing, where multiple services must be composed and must operate simultaneously. +
+
+For details see xref:understanding-orchestration-using-mule.adoc[Implement Orchestration in Mule] documentation.
+
+== Implement Enterprise Integration Patterns
+Implement enterprise integration patterns as solutions for designing, building messaging and integration systems, as well as a common language for teams to use when architecting solutions.
+Mule reduces the effort required when building integrations by implementing the patterns that you use to design solutions. +
```

Comment:
This sentence seems to just repeat parts of the previous sentence, without adding any information.